src/scss/components/navigation/responsive.scss

```diff
@@ -0,0 +1,130 @@
+@media screen and (max-width: 1200px) {
+  .main-navigation__list {
+      margin-right: 0;
+  }
+}
+@media screen and (max-width: 992px) {
+  .main-navigation {
+    position: relative;
+    z-index: 100;
+    &__list {
+      position: absolute;
+      right: -400px;
+      top: 100%;
+      flex-direction: column;
+      align-items: flex-start;
+      transition: right 0.3s ease;
+      box-shadow: 3px 3px 10px 0 rgba(0,0,0, 0.3);
+      height: auto;
+      background: #fff;
+
+      &.active {
+        right: 0;
+      }
+
+    }
+    &__item {
+      position: relative;
+      flex-direction: column;
+      align-items: flex-start;
+      min-width: 200px;
+      margin-right: 0;
+    }
+    &__item-link {
+      padding: 10px 15px;
+    }
+    &__sub-menu {
+      position: static;
+      box-shadow: none;
+      max-height: 0;
+      height: auto;
+      transition: all 0.5s ease;
+      overflow: hidden;
+      visibility: visible;
+      opacity: 1;
+
+      & .main-navigation__item {
+        padding: 0;
+      }
+    }
+    .sub-menu-indicator {
+      position: absolute;
+      right: 15px;
+      top: 5px;
+      padding: 10px;
+      z-index: 110;
+
+      &::after {
+        right: calc(50% - 5px);
+        top: calc(50% - 7px);
+      }
+    }
+    &__item_has-children {
+      &:hover {
+        & .sub-menu-indicator::after {
+          border-color: $brand-color;
+          top: calc(50% - 7px);
+          transform: rotate(45deg);
+        }
+      }
+
+      &.active {
+        & .sub-menu-indicator {
+          &::after {
+            border-color: $main-link-color;
+            top: calc(50% - 3px);
+            transform: rotate(-136deg);
+          }
+        }
+        .main-navigation__sub-menu {
+          max-height: 500px;
+          transition: max-height 1s ease;
+        }
+      }
+    }
+    &__controls {
+      position: relative;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      height: 28px;
```

Comment:
The size of the trigger is controlled by the height of one wrapper and the width of the other.
Here is a suggestion:
.trigger-wrapper {width: 32px; height: 0; padding-top: 100%;} /*Drawing a scalable square*/
.trigger-mark {width: 100%; height: 12.5%;} /* Height is 32px, 12.5% of that is 4px */
.trigger-mark::before, .trigger-mark::after {width: 100%; height: 100%;} /* Since trigger-mark is absolute positioned width and height will be inherited */
This approaches allows to control the whole sizing using one property (width of the wrapper).